.github/workflows/pull-db-tests.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         with:
           go-version: ">=1.20.0"
       - name: Add hosts to /etc/hosts
-        run: echo "127.0.0.1 pgsql ldap minio" | sudo tee -a /etc/hosts
+        run: '[ -f "/.dockerenv" ] || echo "127.0.0.1 pgsql ldap minio" | sudo tee -a /etc/hosts'
```

Comment:
This file is specific to the docker daemon, it will not work in other OCI container runtimes like podman. A slightly better check would be

```bash
[ -e "/.dockerenv" ] || [ -e "/run/.containerenv" ]
```